other/run-examples.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+const { exec } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+const examplesFolder = 'packages/data-point/examples/'
+
+fs.readdir(examplesFolder, (err, files) => {
+  if (err) {
+    console.log(err)
+    return
+  }
+
+  coloredLog('blue', `Executing all examples in ${examplesFolder}`, true)
+
+  files.forEach(file => {
+    const filename = examplesFolder + file
+    exec(`node ${path.resolve(filename)}`, (err, stdout, stderr) => {
+      if (err) {
+        console.error(err)
+        return
+      }
+
+      if (stdout) {
```

Comment:
to keep things clean can u remove any non-error logs, I think at the most would be useful to know how many files were tested and in how much time